mobile/lib/domain/services/store.service.dart

```diff
@@ -0,0 +1,106 @@
+import 'dart:async';
+
+import 'package:immich_mobile/domain/interfaces/store.interface.dart';
+import 'package:immich_mobile/entities/store.entity.dart';
+
+class StoreService {
+  final IStoreRepository _storeRepo;
```

Comment:
I think we don't use abbreviation much, so let's stick with the whole word i.e `_storeRepository`